src/components/menu/MenuHeader.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/variables';
+
+.bdl-MenuHeader {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
```

Comment:
Does this `justify-content: space-between` have an effect?

I'm thinking that because `.bdl-MenuHeader-content` has `flex-grow: 1`, that this possibly wouldn't do anything because there would be no space between `.bdl-MenuHeader-content` and the `CloseButton`.

Reply:
yeah you're right. I just tried it and it's removed now